<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="clearBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="popBackBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Pop Back</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="pushBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="secondInsertBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>81</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
       <widget class="QPushButton" name="firstPushBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Push Back</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="secondIntBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="secondDblBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>39</x>
          <y>1</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>1</y>
      <width>601</width>
      <height>531</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
